packages/terra-brand-footer/examples/test-examples/ActiveBrandFooter.scss

```diff
@@ -0,0 +1,9 @@
+:local {
+  .active-brand-footer {
+    // stylelint-disable selector-max-compound-selectors
```

Comment:
If we can, we should find a way to drop this disable stylelint rule

Reply:
Oh, I see that these styles are styling the nav bar of the footer, which I don't think we should do here. We should instead provide the ability to attach classes to the nav and styles then.

Reply:
These styles are for mocking the active state of the link so we can capture a screenshot of the `:active` state of the links without triggering it through mouse interaction in wdio.

Reply:
Adding to @bjankord  comment, I used these styles in the test example to mock the active, hover state of hyperlinks in the navigation. They are not part of styling the component or site example.

Reply:
Ahhh 😅 Ok! I missed these were mocked styles